CONTRIBUTING.md

```diff
@@ -1,100 +1,239 @@
+
+The [Sanity Studio](https://www.sanity.io/docs/sanity-studio) is an open-source editing environment built with React.
+
+## Branches
+
+- `current`: This contains all the features and fixes included in the latest official release.
+- `next`: This includes everything scheduled for the next, upcoming release.
+
+## Accepted contributions
+
+- Bug reports
+- Feature requests
+- Bug fixes
+- Smaller improvements/enhancements of existing features
+- New features
+- Documentation improvements
+
+New features are unlikely to be accepted and are better developed and published as plugins that extend the existing functionality of the Sanity Studio. See [New features](#new-features) for more info.
+
+### Security issues
+
+If you find a security vulnerability, do **NOT** open an issue.
+
+Email `security@sanity.io` instead.
+
+Any security issues should be submitted directly to `security@sanity.io`. In order to determine whether you are dealing with a security issue, ask yourself these two questions:
+
+- Can I access something that's not mine, or something I shouldn't have access to?
+- Can I disable something for other people?
+
+If the answer to either of those two questions are "yes", then you're probably dealing with a security issue. Note that even if you answer "no" to both questions, you may still be dealing with a security issue, so if you're unsure, just email us at `security@sanity.io`.
+
+### Bug reports
+
+To submit a bug report, check if the issue has already been reported, and if not, please create a new issue and use the provided template. Make sure your report answers these five questions:
+
+- Which versions of Sanity are you using (`sanity versions`)?
+- What operating system are you using?
+- Which versions of Node.js / npm are you running?
+- What did you do?
+- What did you expect to see?
+- What did you see instead?
+
+### Feature Requests or Enhancements
+
+Have you found yourself wishing for a feature that doesn't exist in Sanity? It's likely you are not alone in wanting a particular feature and you should submit a Feature Request. We appreciate and consider all feature requests, and many of the features that Sanity has today have been added because someone saw the need for them.
+
+To submit a Feature Request, please make sure that it doesn't already exist by searching through our [open issues](https://github.com/sanity-io/sanity/issues), before you create a new one. If someone already submitted the same feature requests you wanted to submit, please let us know you also want this feature by reacting with/adding a :+1: on the original comment, or leave a new comment expressing your need/excitement for this feature, or any other relevant information.
+
+If the same Feature Request doesn't already exist, create a new issue and use the Feature Requests template. Be sure to provide all the necessary information about your new feature to help us understand what you are requesting.
+
+# Setting up Sanity locally
```

Comment:
Should we say something about the `symlinkDependencies` script in order to use local Sanity packages in an existing studio?